config.yaml

```diff
@@ -5,119 +5,135 @@ resources:
       - all
     events:                  # List of lifecycle events you want to receive, e.g create, update, delete, error OR all
       - create
+      - update
```

Comment:
@aananthraj lets keep `update` events only for `pod` and `job` resources in the default configurations. I don't think people will be interested in update events of all the resources

Reply:
Sure lets make them available.
I guess, we need that for deployment as well.

Reply:
I would say either deployment or pod. Not both. Otherwise, the user will get redundant notifications

Reply:
sure, then let's make them default for pods and jobs

Reply:
resolved it

Reply:
Thanks. Just a thought, would it make sense to configure update for `deployment` and `statefulset` instead of `pod`. Because if deployment/statefulset has multiple replica of pod, it will send redundant notifications about each pod updates

Reply:
In general, `Deployments` and `Statefulset` are deployed in clusters to manage replicas. hence informed it as
> I guess, we need that for deployment as well.

added update event to `Deployments`, `Statefulset` and `Job` by default.